tools/topology/topology2/cavs/cavs-nocodec.conf

```diff
@@ -228,7 +228,7 @@ Object.Pipeline {
 		}
 
 		Object.Widget.copier.1 {
-			dai_index 1
```

Comment:
Can you please change the commit title and description. Theres's no need to mention CI in the commit messages. Just say "Fix dai_index for SSP 2 DAI copier widget".

Reply:
we should let this one slip @ranj063 and merge if the results are ok to unblock @kv2019i tests

Reply:
agree @plbossart. i've approved already

Reply:
> Theres's no need to mention CI in the commit messages.

It's worse than "no need" actually: if an issue with a wrong `dai_index` can be reproduced only in CI then something is wrong either with CI or with the person trying to reproduce.

Of course this should not block this small commit and the rest of the commit message looks OK, I just want to make clear that CI should never be considered as some kind of parallel universe.